docs/topics/testing-rules.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/rules-development-guide/master.adoc
 
+:_content-type: REFERENCE
```

Comment:
@sbeskin-redhat @stoobie I think this is a procedure that might need to be broken up into several modules. We're telling users what/how to do something.

Reply:
@RichardHoch It does have some elements of a procedure, although there are no precise instructions. But it also gives reference information on rule structure and syntax.

Reply:
@sbeskin-redhat @RichardHoch 
This is REFERENCE info, but my main concern with this module is the vast amount of passive tense.
Try rewriting using imperative as much as possible, and change "should" to "do" (You don't have to literally include the word "do"), tell users what to do.
These modules really need a bit of rewriting.
The first sentence doesn't make sense to me:
`Test rules are created using a process similar to the process for creating a test rule`

This one module should be broken into several modules, all at the same level. `Creating a test rule` should be a peer of the modules that follow it. In fact, I think that the heading "Creating a test rule" should be removed.
Sasha, can you create a separate PR for this? In the meantime, I think that all this content is reference.

Reply:
Will create a separate PR